bot/cogs/utilities.py

```diff
@@ -183,6 +233,77 @@ async def quotes(self, ctx: Context) -> None:
 
         await ctx.send(embed=embed_quote)
 
+    @command(brief="Get the info on the user specified, defaults to the command author",
+             usage=".userinfo [user (optional)]",
+             aliases=('ui', 'user', 'useri', 'uinfo'))
+    async def userinfo(self, ctx: Context, user: Member = None) -> None:
+        """Shows an embed containing basic info on the user."""
+        user = ctx.author if not user else user
+        spotify_emoji = '<:spotify:785113868543852584>'
+        user_badges = []
+        roles = []
+        statuses = []
+        is_bot = "Bot: :x:"
+
+        create_time = ', '.join(self.get_timedelta(datetime.datetime.utcnow(), user.created_at))
+        joined_time = ', '.join(self.get_timedelta(datetime.datetime.utcnow(), user.joined_at))
+
+        flags = user.public_flags
+
+        info_emb = Embed(color=user.color,
+                         title=str(user))
+        info_emb.set_thumbnail(url=user.avatar_url)
+
+        for flag_ in flags.all():
+            if str(flag_.name) in BADGES:
+                user_badges.append(BADGES[str(flag_.name)])
+
+        if user.is_avatar_animated():
+            user_badges.append(BADGES['nitro'])
+
+        for role in user.roles[1:]:
+            roles.append(f"<@&{role.id}>")
+
+        if user.bot:
+            is_bot = "Bot: :white_check_mark:"
+
+        statuses.append(f"{STATUSES[user.mobile_status]} Mobile Client")
+        statuses.append(f"{STATUSES[user.web_status]} Web Client")
+        statuses.append(f"{STATUSES[user.desktop_status]} Desktop Client")
+
+        roles = ', '.join(roles)
+        user_badges = " ".join(user_badges)
+        statuses = '\n'.join(statuses)
+        info_emb.description = user_badges
+        men, id_, nick = user.mention, user.id, user.nick
+
+        base_activity = user.activity
+        if base_activity:
+            activity = ACTIVITIES[base_activity.type] + base_activity.name
+        else:
+            activity = "No activity is being done"
+
+        if isinstance(base_activity, Spotify):
+            activity = "\n".join([f'{spotify_emoji} {activity[12:]}',
+                                  f"**Song**: {base_activity.title}",
+                                  f"**Artist**: {base_activity.artist}",
+                                  f"**Album**: {base_activity.album}"])
+
+        info_emb.add_field(name='User Information',
+                           value=f"Created: {create_time} ago\nProfile: {men}\nID: {id_}\n{is_bot}",
+                           inline=False)
+        info_emb.add_field(name="Guild Profile",
+                           value=f"Joined: {joined_time} ago\nNick: {nick}\nRoles: {roles}",
+                           inline=False)
+        info_emb.add_field(name="Status",
+                           value=statuses,
+                           inline=False)
+        info_emb.add_field(name="Activity",
+                           value=activity,
+                           inline=False)
```

Comment:
You can combine all add_field calls into a for loop. The information would be stored in a dictionary for ease of editibility in the future.
```suggestion
        user_info = {
                'User Information': f'Created: {create_time} ago\nProfile: {men}\nID: {id_}\n{is_bot}',
                'Guild Profile': f'Joined: {joined_time} ago\nNick: {nick}\nRoles: {roles}',
                'Status': statuses,
                'Activity': activity
       }
       for k, v in user_info.items():
                info_emb.add_field(name=k, value=v, inline=False)
```